src/services/apiService.ts

```diff
@@ -27,6 +27,14 @@ export class ApiService {
         });
     }
 
+    public getBlockchainStateByPublicKey(publicKey: string): Promise<State> {
+        return new Promise(async (resolve, reject) => {
+            Axios.get<State>(this.getStatesRoute() + "/" + publicKey)
```

Comment:
Please use string interpolation instead of string concatenation.
```suggestion
            Axios.get<State>(`${this.getStatesRoute()}/${publicKey})
```